tools/engine_tool/lib/src/commands/lint_command.dart

```diff
@@ -79,7 +84,7 @@ final class LintCommand extends CommandBase {
     // TODO(loic-sharma): Relax this restriction.
     if (environment.platform.isWindows) {
       environment.logger
-          .error('lint command is not supported on Windows (for now).');
+          .fatal('lint command is not supported on Windows (for now).');
```

Comment:
Since this immediately prints the log and returns an error code to `main()` anyway, the only difference from `logger.error()` here is the exception with the stack trace. That seems desirable if the idea is that we'll only use `fatal()` to signal the program is in a state that should be impossible, or something that isn't implemented yet, like this case. The stack trace will help the user file a bug that we can do something with better than just a single log message.

If that's the intended usage, can we put that in a comment on the method in the logger? That is, we shouldn't be throwing an exception for an error that the user is expected to fix themselves.

Reply:
Done.